tutorials/shaders/shader_reference/shading_language.rst

```diff
@@ -722,55 +722,61 @@ When vec_type (float), vec_int_type, vec_uint_type, vec_bool_type nomenclature i
 | vec_type **inversesqrt** (vec_type x)                                  | Inverse square root                                           |
 +------------------------------------------------------------------------+---------------------------------------------------------------+
-| vec_type **abs** (vec_type x)                                          | Absolute                                                      |
+| vec_type **abs** (vec_type x)                                          | Absolute value (returns positive value if negative)           |
 +------------------------------------------------------------------------+---------------------------------------------------------------+
-| ivec_type **abs** (ivec_type x)                                        | Absolute                                                      |
+| ivec_type **abs** (ivec_type x)                                        | Absolute value (returns positive value if negative)           |
 +------------------------------------------------------------------------+---------------------------------------------------------------+
-| vec_type **sign** (vec_type x)                                         | Sign                                                          |
+| vec_type **sign** (vec_type x)                                         | Sign (returns ``1.0`` if positive, ``-1.0`` if negative,      |
+|                                                                        | ``0.0`` if zero)                                              |
 +------------------------------------------------------------------------+---------------------------------------------------------------+
-| ivec_type **sign** (ivec_type x)                                       | Sign                                                          |
+| ivec_type **sign** (ivec_type x)                                       | Sign (returns ``1`` if positive, ``-1`` if negative,          |
+|                                                                        | ``0`` if zero)                                                |
 +------------------------------------------------------------------------+---------------------------------------------------------------+
-| vec_type **floor** (vec_type x)                                        | Floor                                                         |
+| vec_type **floor** (vec_type x)                                        | Round to the integer below                                    |
 +------------------------------------------------------------------------+---------------------------------------------------------------+
-| vec_type **round** (vec_type x)                                        | Round                                                         |
+| vec_type **round** (vec_type x)                                        | Round away from zero                                          |
```

Comment:
I don't know if this is vendor-specific. Maybe "Round to the nearest integer" is a safer choice here. @cc clayjohn